ext/on-modify.timewarrior

```diff
@@ -42,6 +42,18 @@ old = json.loads(sys.stdin.readline())
 new = json.loads(sys.stdin.readline())
 print(json.dumps(new))
 
+def get_timew_name_from_json(json_obj):
```

Comment:
I would rename the function to `extract_timew_tags_from_json` or `extract_timew_tags_from` to better reflect its purpose.